plugin/filestyle.vim

```diff
@@ -406,5 +421,6 @@ if !exists('g:filestyle_plugin')
   command! FileStyleDeactivate call FileStyleDeactivate()
   command! FileStyleCheck call FileStyleCheck()
   command! FileStyleFix call FileStyleFix()
+  command! FileStyleToggle call FileStyleToggle()
```

Comment:
In order to keep the consistency, this line should be moved up just after FileStyleDisable.